docs/pages/admin-guides/access-controls/sso/gitlab.mdx

```diff
@@ -162,7 +162,7 @@ The developer role:
 kind: role
 version: v5
 metadata:
-  name: dev
+  name: access
```

Comment:
We can remove this `3/4` step? `access` is already a preset role, the system wont let you create a new one.